sycl/test/check_device_code/matrix/matrix-nvptx-bfloat16-test.cpp

```diff
@@ -0,0 +1,200 @@
+// REQUIRES: cuda
+
+// RUN: %clangxx -fsycl-device-only -fsycl-targets=nvptx64-nvidia-cuda -Xsycl-target-backend --cuda-gpu-arch=sm_80 -DSYCL_EXT_ONEAPI_MATRIX=3 -S -Xclang -emit-llvm %s -o -| FileCheck %s
+
+#include <CL/sycl.hpp>
```

Comment:
Write new tests in SYCL 2020
```suggestion
#include <sycl/sycl.hpp>
```